lms/djangoapps/instructor/tests/test_enrollment.py

```diff
@@ -422,15 +420,16 @@ def test_delete_submission_scores(self, _mock_send_signal, mock_set_receiver, mo
         sub_api.set_score(submission['uuid'], 1, 2)
 
         # Delete student state using the instructor dash
+        mock_send_signal.reset_mock()
         reset_student_attempts(
             self.course_key, user, problem_location,
             requesting_user=user,
             delete_module=True,
         )
 
         # Make sure our grades signal receivers handled the reset properly
-        mock_set_receiver.assert_not_called()
-        mock_reset_receiver.assert_called_once()
```

Comment:
More assertions not present in mock 1.0.1.  Same issue with mocking signal handlers noted above.